airbyte-ci/connectors/metadata_service/orchestrator/orchestrator/assets/registry_entry.py

```diff
@@ -246,6 +266,9 @@ def metadata_to_registry_entry(metadata_entry: LatestMetadataEntry, override_reg
     # Add Dependency information
     overridden_metadata_data["packageInfo"] = apply_package_info_fields(overridden_metadata_data, metadata_entry)
 
+    # Add language field
+    overridden_metadata_data = apply_language_field(overridden_metadata_data)
```

Comment:
This is great but we should update our registry type as well 
https://github.com/airbytehq/airbyte/blob/e519f0fc9674afe8af74ee2a8d4e8f9dc05c411e/airbyte-ci/connectors/metadata_service/lib/metadata_service/models/generated/ConnectorRegistryV0.py

Reply:
Ah yes, thank you!!